docs/setup/appstore.md

```diff
@@ -0,0 +1,26 @@
+# App Store
+
+**Setup :material-menu-right: App Store**
+
+The **ConnexCS App Store** lets you discover, download, and install applications on your user account.
+
+The Apps created using the IDE when published and are available for use in the App Store.
+
+The Apps can be created by the other customers or developers and can be used by ConnexCS's customers.
```

Comment:
The Apps can be created by other customers or developers and used by ConnexCS's customers.

Reply:
done